package.json

```diff
@@ -36,7 +36,7 @@
     "test:debug": "mocha debug test/runner.js --reporter spec"
   },
   "dependencies": {
-    "broccoli-caching-writer": "^0.5.4",
+    "broccoli-caching-writer": "^1.1.0",
```

Comment:
might be, its possible that `pod` also causes some issues,

Im not apposed to your change, but i want to be able to demonstrate the problem myself. As it feels like potentially a patch of the symptom not the problem.